Add PPRV cloud docs for public beta (#446)
hashicorp/terraform-docs-common: website/docs/cloud-docs/policy-enforcement/policy-results.mdx

Reviewer: @mrinalirao I think we may need to rewrite the policy results section. The second sentence states:

```
Terraform Cloud displays the results of policy checks in the UI for each run.
```

We could probably rewrite it as follows so that it applies to checks and evaluations:

```
Terraform Cloud displays the policy enforcement results in the UI for each run
```

## Policy Evaluation Run Stages

Terraform Cloud only evaluates policies for successful plans. Terraform Cloud evaluates Sentinel and OPA policy sets separately and at different points in the run.
- Sentinel policy checks occur after Terraform completes the plan and after both [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) and [cost estimation](https://terraform.io/cloud-dodcs/cost-estimation). This order lets you write Sentinel policies to restrict costs based on the data in the cost estimates.

> -> **Note**: Sentinel policy evaluations are in beta

Reviewer: ```suggestion

-> **Note**: Sentinel policy evaluations are in beta
```

For Sentinel only, you can also override `soft-mandatory` policies with the Terraform CLI. Run the `terraform apply` command and then enter `override` when prompted.
-> **Note:** Terraform Cloud does not allow policy overrides for [no-operation plans containing no infrastructure changes](/terraform/cloud-docs/run/modes-and-options#allow-empty-apply), unless you choose the **Allow empty apply** option when starting the run. 

### Sentinel

#### Policy checks

> Policies with an `advisory` enforcement level never stop runs. If they fail, Terraform Cloud displays a warning in the policy results and the run continues.

Reviewer: ```suggestion
Policies with an `advisory` enforcement level does not block a run. If they fail, Terraform Cloud displays a warning in the policy results and the run continues.
```

You can override `soft-mandatory` policies to allow the run to continue. Overriding failed policies on a run does not affect policy evaluations on future runs in that workspace.

You cannot override `hard-mandatory` policies, and all of these policies must pass for the run to continue.

#### Policy evaluations
-> **Note:** This feature is in beta.

> Policies with an `advisory` enforcement level never stop runs. If they fail, Terraform Cloud displays a warning in the policy results and the run continues.

Reviewer: ```suggestion
Policies with an `advisory` enforcement level do not block a run. If they fail, Terraform Cloud displays a warning in the policy results and the run continues.
```

> When Sentinel policies are run as policy evaluations (if opting in to the `enhanced` workflow), then internally both `soft-mandatory` as well as `hard-mandatory` enforcement_levels are converted to `mandatory` enforcement level (similar to OPA).

Reviewer: ```suggestion
When running Sentinel policies as policy evaluations (if you opt into the 'enhanced' workflow), both 'soft-mandatory' and 'hard-mandatory' enforcement levels are internally converted to 'mandatory' enforcement level, similar to OPA.
```

Reviewer: I have rewritten for clarity

> When running Sentinel policies as policy evaluations (if you opt into the `enhanced` workflow), both `soft-mandatory` and `hard-mandatory` enforcement levels are internally converted to `mandatory` enforcement level, similar to OPA.

Reviewer 2: We can remove some text and make the intent more obvious

```suggestion
When running Sentinel policies as policy evaluations, `soft-mandatory` and `hard-mandatory` enforcement levels are internally converted to `mandatory` enforcement level.
```
